.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -47,12 +47,23 @@ Enter your extended release note in the block below. If the PR requires addition
 
 <!--
 This section can be blank if this pull request does not require a release note.
-Please use the following format for linking documentation or pass the
-section below:
+
+The Kubernetes Release Notes website (https://relnotes.k8s.io) is able to
+provide additional documentation to the end users, which should help them to
+be able to provide additional documentation to the end users about the changes
+in this PR.
```

Comment:
Could you clean up this sentence a bit, I think it's saying the same thing twice? It should also go under the release notes heading staring on line 36, not on the "additional documentation" heading.

Reply:
> I'm not entirely sure we need to mention relnotes in the template - if the actual process for adding a release note is no different for the user than before, I don't think we need to add it here.
> 
> The [document ](https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md) on release notes should be updated, however.

It might still be worth adding a small hint that external references to KEPs etc should link to a specific commit rather than master and fleshing out further instructions details in the release notes doc. 
Something like:
```
For KEPs or other supporting documentation, please link to a specific commit
and avoid linking directly to master. See <link> for further guidelines.
```

Reply:
> Could you clean up this sentence a bit, I think it's saying the same thing twice? It should also go under the release notes heading staring on line 36, not on the "additional documentation" heading.

I removed the section entirely, because I would like to pick up the discussion if its needed at all or not. WDYT?

Reply:
yes! my main concern is honestly that we document the relnotes in the relevant doc, which we link to on top!